### DISCUSSION TITLE
How can I search a specific folder without having to type out the whole path?

### DISCUSSION THREAD

**Discussion by ReaderGuy42** [Ulauncher/Ulauncher#1263] [Q&A]
I have a folder in my Documents directory called `Dictionary`. How can I search this particular folder, e.g. by prefacing the search with `dict` (similar to `ddg`) and then only search that one folder. 

The only way I've found so far is typing out the old path, but there must be an easier way.

I've also tried adding a shortcut with the Query: `/home/user/Documents/Dictionary/` but that doesn't do anything.

Alternately, I use Obsidian.md and keep my dictionary entries as a vault there. In a browser URL bar I can use this to launch specific notes in Obsidian: `obsidian://advanced-uri?vault=Dictionary&filepath=Entries/flower`, with `flower` being the entry that's supposed to open. 

From a browser window this works. But not from ulauncher. 

Advice appreciated!
Thanks :)

**Comment by friday** [comment]
Ulauncher doesn't support built in search (at least not yet). You need an extension for that. Try https://github.com/hillaryychan/ulauncher-fzf or https://github.com/friday/ulauncher-locate. ulauncher-fzf lets you specify a base directory. My ulauncher-locate extension doesn't do that, but it's simple and you can fork and make your own that searches your folder.
